non-conversational-agent/src/agent_server/server.py

```diff
@@ -39,6 +39,8 @@ def decorator(func: Callable):
 
     return decorator
 
+def get_invoke_function():
+    return _invoke_function
 
```

Comment:
note: adding accessor method to retrieve invoke function from agent_server since this is currently stored as private variable -> can you confirm it's alright to make this accessible to the developer @bbqiu ?